### PR TITLE
fix lighthouse url report

### PR DESCRIPTION
**Pull request by FeliceGeracitano**
the regex was stripping off `m.` or `url parameters`, we need to keep all the url info to view the right report in grafana

### CODE REVIEW

File: kubernetes/local/inspector-lighthouse.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
             - name: DB_HOST
               value: influxdb
             - name: DB_PORT
-              value: "8086"
+              value: '8086'
```
Comment (rbelling) [inline kubernetes/local/inspector-lighthouse.yaml]: is this change needed?

Comment (FeliceGeracitano) [reply kubernetes/local/inspector-lighthouse.yaml]: changes from prettier 'singleQuote' = true
